src/dbnode/storage/index.go

```diff
@@ -194,7 +208,8 @@ func newNamespaceIndexWithOptions(
 	idx := &nsIndex{
 		state: nsIndexState{
 			runtimeOpts: nsIndexRuntimeOptions{
-				insertMode: indexOpts.InsertMode(), // FOLLOWUP(prateek): wire to allow this to be tweaked at runtime
+				insertMode:            newIndexOpts.opts.IndexOptions().InsertMode(), // FOLLOWUP(prateek): wire to allow this to be tweaked at runtime
```

Comment:
This can just be `indexOpts.InsertMode()` still yeah?

Reply:
sure thing

Reply:
done